config.c

```diff
@@ -636,6 +676,15 @@ int load_config_file(struct mako_config *config, char *config_arg) {
 			line[strlen(line) - 1] = '\0';
 		}
 
+		if (line[0] == '{' && line[strlen(line) - 1] == '}') {
```

Comment:
I'm going to have to ponder how I feel about adding a new type of section for this. I will admit no alternatives immediately spring to mind. Can you update the manpage with some examples of this so we can get a better idea of how it will feel?